clients/src/main/java/org/apache/kafka/clients/consumer/KafkaConsumer.java

```diff
@@ -1365,7 +1365,7 @@ public void commitSync(Duration timeout) {
             maybeThrowInvalidGroupIdException();
             if (!coordinator.commitOffsetsSync(subscriptions.allConsumed(), time.timer(timeout))) {
                 throw new TimeoutException("Timeout of " + timeout.toMillis() + "ms expired before successfully " +
-                        "committing the current consumed offsets");
+                        "committing the current consumed offsets. Try tuning 'default.api.timeout.ms' option");
```

Comment:
I feel a better instruction needs a tuning direction, maybe `Try increasing 'default.api.timeout.ms' as appropriate`

Reply:
yep, the direction is a little obscure, so I modified the direction.